inc/wplf-register-form.php

```diff
@@ -0,0 +1,161 @@
+<?php
+// A function to register a form in the code.
+// 
+// You would probably want to use this code inside a plugin 
+// activation hook or something similar so that it wouldn't
+// be ran on every page load.
+// 
+// Note that if you change the contents of a field registered in
+// code in the admin panel, your changes will get overridden
+// whenever the register function is ran again.
+// 
+// Usage:
+// 
+// function register_libre_forms() {
+//     register_libre_form( "slug", [
+//         "title" => "Name",
+//         "form_html" => "form's html",
+//         "emails" => [
+//             "first@recipient.com" => [
+//                 "title" => "Title",
+//                 "content" => "Content."
+//             ],
+//             "%email%" => [
+//                 "title" => "This mail goes to %email%",
+//                 "content" => "And contains %content%"
+//             ],
+//             "with@defaulttemplate.com" => []
+//         ],
+//         "title_format" => "A submission from %email%",
+//         "thank_you" => "thank you html"
+//     ]);
+// }
+
+function register_libre_form( $name, $args ) {
+    // Throw an error if the function is not properly called
+    if ( ! post_type_exists( 'wplf-form' ) ) {
```

Comment:
Just a small formatting thing, but really bothers me. :) Can you change the tabsize to 2 like all the other files, for this one as well?